spec/requests/api/v1/docs/display_ads_spec.rb

```diff
@@ -47,31 +49,8 @@
 
         produces "application/json"
         consumes "application/json"
-        parameter name: :display_ad, in: :body, schema: {
-          type: :object,
-          properties: {
-            name: { type: :string, description: "For internal use, helps distinguish ads from one another" },
-            body_markdown: { type: :string, description: "The text (in markdown) of the ad (required)" },
-            approved: { type: :boolean, description: "Ad must be both published and approved to be in rotation" },
-            published: { type: :boolean, description: "Ad must be both published and approved to be in rotation" },
-            organization_id: { type: :integer, description: "Identifies the organization to which the ad belongs" },
-            display_to: { type: :string, enum: DisplayAd.display_tos.keys, default: "all",
-                          description: "Potentially limits visitors to whom the ad is visible" },
-            type_of: { type: :string, enum: DisplayAd.type_ofs.keys, default: "in_house",
-                       description: <<~DESCRIBE
-                         Types of the billboards:
-                         in_house (created by admins),
-                         community (created by an entity, appears on entity's content),
-                         external ( created by an entity, or a non-entity, can appear everywhere)
-                       DESCRIBE
-                    },
-            placement_area: { type: :string, enum: DisplayAd::ALLOWED_PLACEMENT_AREAS,
-                              description: "Identifies which area of site layout the ad can appear in" },
-            tag_list: { type: :string, description: "Tags on which this ad can be displayed (blank is all/any tags)" },
-            creator_id: { type: :integer, description: "Identifies the user who created the ad." }
-          },
-          required: %w[name body_markdown placement_area]
-        }
+        parameter name: :display_ad, in: :body, schema: { type: :object,
```

Comment:
Nothing to do with your work in this file, just the fact that this file exists at all is pretty neat - I love this "autogenerate the JSONSchema from real enum hashes" approach.